docs/contributing.html

```diff
@@ -331,10 +331,11 @@ <h2 id="toc-title">On this page</h2>
   <li><a href="#adding-a-new-format" id="toc-adding-a-new-format" class="nav-link" data-scroll-target="#adding-a-new-format">1. Adding a new format</a></li>
   <li><a href="#modify-or-add-to-an-existing-format" id="toc-modify-or-add-to-an-existing-format" class="nav-link" data-scroll-target="#modify-or-add-to-an-existing-format">2. Modify or add to an existing format</a></li>
   <li><a href="#adding-a-cookbook" id="toc-adding-a-cookbook" class="nav-link" data-scroll-target="#adding-a-cookbook">3. Adding a cookbook</a></li>
-  </ul></li>
   <li><a href="#optional-update-slides" id="toc-optional-update-slides" class="nav-link" data-scroll-target="#optional-update-slides">4. (Optional) Update slides</a></li>
   <li><a href="#add-yourself-to-the-list-of-authors-on-the-welcome-page" id="toc-add-yourself-to-the-list-of-authors-on-the-welcome-page" class="nav-link" data-scroll-target="#add-yourself-to-the-list-of-authors-on-the-welcome-page">5. Add yourself to the list of authors on the Welcome page!</a></li>
   <li><a href="#once-your-pr-is-approved-merge-away." id="toc-once-your-pr-is-approved-merge-away." class="nav-link" data-scroll-target="#once-your-pr-is-approved-merge-away.">6. Once your PR is approved, merge away.</a></li>
+  </ul></li>
```

Comment:
Wait, why do we have a `contributing.html` checked in to Git? Isn't this redundant with `contributing.qmd`?

Reply:
I'm deploying the site by [rendering to docs](https://quarto.org/docs/publishing/github-pages.html#render-to-docs), which creates the static html for the site and publishes it in the repo. LMK if you have opinions on a better way to do this. I can see how this approach might confuse people who want to contribute to the guide.

Reply:
I think quarto mentions that option first because it's less confusing to Git newcomers who don't want to think about multiple branches. But in my opinion it's preferable to not have HTML output checked in to git because you'll have many "built" changes for only a few lines of source code changes. In that case the lower docs on ["Publish command"](https://quarto.org/docs/publishing/github-pages.html#publish-command) are preferable.

See also https://github.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/issues/58

Reply:
@kylebarron I think I've fixed this. Please take a look and let me know if you're ok merging this.

Reply:
lgtm!